pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -310,21 +310,22 @@ def test_to_datetime_tz_psycopg2(self):
     def test_datetime_bool(self):
         # GH13176
         with self.assertRaises(TypeError):
-            to_datetime(False)
-        self.assertTrue(to_datetime(False, errors="coerce") is NaT)
-        self.assertEqual(to_datetime(False, errors="ignore"), False)
+            to_datetime(False, unit='ns')
```

Comment:
ideally you would parametrize this on units to test all the units, see https://github.com/pandas-dev/pandas/blob/master/pandas/tests/indexes/datetimes/test_tools.py#L1520

(move the units fixture to the top of the file)